### ISSUE TITLE
2025-03-21, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#116]
# Today's Arxiv Papers

(1) [Stop Overthinking: A Survey on Efficient Reasoning for Large Language Models](http://arxiv.org/abs/2503.16419v1), 2025-03-20

(2) [Survey on Evaluation of LLM-based Agents](http://arxiv.org/abs/2503.16416v1), 2025-03-20

(3) [M3: 3D-Spatial MultiModal Memory](http://arxiv.org/abs/2503.16413v1), 2025-03-20

(4) [The Emperor's New Clothes in Benchmarking? A Rigorous Examination of Mitigation Strategies for LLM Benchmark Data Contamination](http://arxiv.org/abs/2503.16402v1), 2025-03-20

(5) [Exploring the Hidden Reasoning Process of Large Language Models by Misleading Them](http://arxiv.org/abs/2503.16401v1), 2025-03-20

(6) [Deconstructing Long Chain-of-Thought: A Structured Reasoning Optimization Framework for Long CoT Distillation](http://arxiv.org/abs/2503.16385v1), 2025-03-20

(7) [LaPIG: Cross-Modal Generation of Paired Thermal and Visible Facial Images](http://arxiv.org/abs/2503.16376v1), 2025-03-20

(8) [CaKE: Circuit-aware Editing Enables Generalizable Knowledge Learners](http://arxiv.org/abs/2503.16356v1), 2025-03-20

(9) [LLM Braces: Straightening Out LLM Predictions with Relevant Sub-Updates](http://arxiv.org/abs/2503.16334v1), 2025-03-20

(10) [OmniGeo: Towards a Multimodal Large Language Models for Geospatial Artificial Intelligence](http://arxiv.org/abs/2503.16326v1), 2025-03-20

(11) [Issue2Test: Generating Reproducing Test Cases from Issue Reports](http://arxiv.org/abs/2503.16320v1), 2025-03-20

(12) [Bridging Technology and Humanities: Evaluating the Impact of Large Language Models on Social Sciences Research with DeepSeek-R1](http://arxiv.org/abs/2503.16304v1), 2025-03-20

(13) [Uni-3DAR: Unified 3D Generation and Understanding via Autoregression on Compressed Spatial Tokens](http://arxiv.org/abs/2503.16278v1), 2025-03-20

(14) [Chain of Functions: A Programmatic Pipeline for Fine-Grained Chart Reasoning Data](http://arxiv.org/abs/2503.16260v1), 2025-03-20

(15) [Plug-and-Play 1.x-Bit KV Cache Quantization for Video Large Language Models](http://arxiv.org/abs/2503.16257v1), 2025-03-20

(16) [Fin-R1: A Large Language Model for Financial Reasoning through Reinforcement Learning](http://arxiv.org/abs/2503.16252v1), 2025-03-20

(17) [Reinforcement Learning for Reasoning in Small LLMs: What Works and What Doesn't](http://arxiv.org/abs/2503.16219v1), 2025-03-20

(18) [MathFusion: Enhancing Mathematic Problem-solving of LLM through Instruction Fusion](http://arxiv.org/abs/2503.16212v1), 2025-03-20

(19) [Affective Polarization Amongst Swedish Politicians](http://arxiv.org/abs/2503.16193v1), 2025-03-20

(20) [Large Language Models for Water Distribution Systems Modeling and Decision-Making](http://arxiv.org/abs/2503.16191v1), 2025-03-20

(21) [CLS-RL: Image Classification with Rule-Based Reinforcement Learning](http://arxiv.org/abs/2503.16188v1), 2025-03-20

(22) [CodeReviewQA: The Code Review Comprehension Assessment for Large Language Models](http://arxiv.org/abs/2503.16167v1), 2025-03-20

(23) [SpeCache: Speculative Key-Value Caching for Efficient Generation of LLMs](http://arxiv.org/abs/2503.16163v1), 2025-03-20

(24) [Towards Lighter and Robust Evaluation for Retrieval Augmented Generation](http://arxiv.org/abs/2503.16161v1), 2025-03-20

(25) [Automatically Generating Chinese Homophone Words to Probe Machine Translation Estimation Systems](http://arxiv.org/abs/2503.16158v1), 2025-03-20

(26) [Only a Little to the Left: A Theory-grounded Measure of Political Bias in Large Language Models](http://arxiv.org/abs/2503.16148v1), 2025-03-20

(27) [Unify and Triumph: Polyglot, Diverse, and Self-Consistent Generation of Unit Tests with LLMs](http://arxiv.org/abs/2503.16144v1), 2025-03-20

(28) [MKG-Rank: Enhancing Large Language Models with Knowledge Graph for Multilingual Medical Question Answering](http://arxiv.org/abs/2503.16131v1), 2025-03-20

(29) [The Impact of Revealing Large Language Model Stochasticity on Trust, Reliability, and Anthropomorphization](http://arxiv.org/abs/2503.16114v1), 2025-03-20

(30) [Cultural Alignment in Large Language Models Using Soft Prompt Tuning](http://arxiv.org/abs/2503.16094v1), 2025-03-20

(31) [Tuning LLMs by RAG Principles: Towards LLM-native Memory](http://arxiv.org/abs/2503.16071v1), 2025-03-20

(32) [Meta-Learning Neural Mechanisms rather than Bayesian Priors](http://arxiv.org/abs/2503.16048v1), 2025-03-20

(33) [Incomplete Utterance Rewriting with Editing Operation Guidance and Utterance Augmentation](http://arxiv.org/abs/2503.16043v1), 2025-03-20

(34) [GreenIQ: A Deep Search Platform for Comprehensive Carbon Market Analysis and Automated Report Generation](http://arxiv.org/abs/2503.16041v1), 2025-03-20

(35) [Evaluating Test-Time Scaling LLMs for Legal Reasoning: OpenAI o1, DeepSeek-R1, and Beyond](http://arxiv.org/abs/2503.16040v1), 2025-03-20

(36) [Hybrid-Level Instruction Injection for Video Token Compression in Multi-modal Large Language Models](http://arxiv.org/abs/2503.16036v1), 2025-03-20

(37) [The Lighthouse of Language: Enhancing LLM Agents via Critique-Guided Improvement](http://arxiv.org/abs/2503.16024v1), 2025-03-20

(38) [BadToken: Token-level Backdoor Attacks to Multi-modal Large Language Models](http://arxiv.org/abs/2503.16023v1), 2025-03-20

(39) [Corrective In-Context Learning: Evaluating Self-Correction in Large Language Models](http://arxiv.org/abs/2503.16022v1), 2025-03-20

(40) [Autonomous AI imitators increase diversity in homogeneous information ecosystems](http://arxiv.org/abs/2503.16021v1), 2025-03-20

(41) [GraspCoT: Integrating Physical Property Reasoning for 6-DoF Grasping under Flexible Language Instructions](http://arxiv.org/abs/2503.16013v1), 2025-03-20

(42) ["This could save us months of work" -- Use Cases of AI and Automation Support in Investigative Journalism](http://dx.doi.org/10.1145/3706599.3719856), 2025-03-20

(43) [ECKGBench: Benchmarking Large Language Models in E-commerce Leveraging Knowledge Graph](http://arxiv.org/abs/2503.15990v1), 2025-03-20

(44) [GAN-enhanced Simulation-driven DNN Testing in Absence of Ground Truth](http://arxiv.org/abs/2503.15953v1), 2025-03-20

(45) [Adaptive Group Policy Optimization: Towards Stable Training and Token-Efficient Reasoning](http://arxiv.org/abs/2503.15952v1), 2025-03-20

(46) [From Chaos to Order: The Atomic Reasoner Framework for Fine-grained Reasoning in Large Language Models](http://arxiv.org/abs/2503.15944v1), 2025-03-20

(47) [Advancing Mobile GUI Agents: A Verifier-Driven Approach to Practical Deployment](http://arxiv.org/abs/2503.15937v1), 2025-03-20

(48) [Towards Automatic Continual Learning: A Self-Adaptive Framework for Continual Instruction Tuning](http://arxiv.org/abs/2503.15924v1), 2025-03-20

(49) [SPIN: Accelerating Large Language Model Inference with Heterogeneous Speculative Models](http://arxiv.org/abs/2503.15921v1), 2025-03-20

(50) [From Structured Prompts to Open Narratives: Measuring Gender Bias in LLMs Through Open-Ended Storytelling](http://arxiv.org/abs/2503.15904v1), 2025-03-20

(51) [Parameters vs. Context: Fine-Grained Control of Knowledge Reliance in Language Models](http://arxiv.org/abs/2503.15888v1), 2025-03-20

(52) [DocVideoQA: Towards Comprehensive Understanding of Document-Centric Videos through Question Answering](http://arxiv.org/abs/2503.15887v1), 2025-03-20

(53) [Enhancing Zero-Shot Image Recognition in Vision-Language Models through Human-like Concept Guidance](http://arxiv.org/abs/2503.15886v1), 2025-03-20

(54) [Human or LLM? A Comparative Study on Accessible Code Generation Capability](http://arxiv.org/abs/2503.15885v1), 2025-03-20

(55) [DeepPsy-Agent: A Stage-Aware and Deep-Thinking Emotional Support Agent System](http://arxiv.org/abs/2503.15876v1), 2025-03-20

(56) [MASH-VLM: Mitigating Action-Scene Hallucination in Video-LLMs through Disentangled Spatial-Temporal Representations](http://arxiv.org/abs/2503.15871v1), 2025-03-20

(57) [TruthLens: Explainable DeepFake Detection for Face Manipulated and Fully Synthetic Data](http://arxiv.org/abs/2503.15867v1), 2025-03-20

(58) [DroidTTP: Mapping Android Applications with TTP for Cyber Threat Intelligence](http://arxiv.org/abs/2503.15866v1), 2025-03-20

(59) [Uncertainty Quantification and Confidence Calibration in Large Language Models: A Survey](http://arxiv.org/abs/2503.15850v1), 2025-03-20

(60) [Entropy-based Exploration Conduction for Multi-step Reasoning](http://arxiv.org/abs/2503.15848v1), 2025-03-20

(61) [Automatic Generation of Safety-compliant Linear Temporal Logic via Large Language Model: A Self-supervised Framework](http://arxiv.org/abs/2503.15840v1), 2025-03-20

(62) [Enhancing LLM Code Generation with Ensembles: A Similarity-Based Selection Approach](http://arxiv.org/abs/2503.15838v1), 2025-03-20

(63) [Fùxì: A Benchmark for Evaluating Language Models on Ancient Chinese Text Understanding and Generation](http://arxiv.org/abs/2503.15837v1), 2025-03-20

(64) [A Vision Centric Remote Sensing Benchmark](http://arxiv.org/abs/2503.15816v1), 2025-03-20

(65) [Attention Pruning: Automated Fairness Repair of Language Models via Surrogate Simulated Annealing](http://arxiv.org/abs/2503.15815v1), 2025-03-20

(66) [ChatGPT and U(X): A Rapid Review on Measuring the User Experience](http://arxiv.org/abs/2503.15808v1), 2025-03-20

(67) [Video-VoT-R1: An efficient video inference model integrating image packing and AoE architecture](http://arxiv.org/abs/2503.15807v1), 2025-03-20

(68) [DNA Bench: When Silence is Smarter -- Benchmarking Over-Reasoning in Reasoning LLMs](http://arxiv.org/abs/2503.15793v1), 2025-03-20

(69) [Grammar and Gameplay-aligned RL for Game Description Generation with LLMs](http://arxiv.org/abs/2503.15783v1), 2025-03-20

(70) [AutoDrive-QA- Automated Generation of Multiple-Choice Questions for Autonomous Driving Datasets Using Large Vision-Language Models](http://arxiv.org/abs/2503.15778v1), 2025-03-20

(71) [Detecting LLM-Written Peer Reviews](http://arxiv.org/abs/2503.15772v1), 2025-03-20

(72) [Dialogic Learning in Child-Robot Interaction: A Hybrid Approach to Personalized Educational Content Generation](http://arxiv.org/abs/2503.15762v1), 2025-03-20

(73) [GraPLUS: Graph-based Placement Using Semantics for Image Composition](http://arxiv.org/abs/2503.15761v1), 2025-03-20

(74) [AutoRedTeamer: Autonomous Red Teaming with Lifelong Attack Integration](http://arxiv.org/abs/2503.15754v1), 2025-03-20

(75) [Using Language Models to Decipher the Motivation Behind Human Behaviors](http://arxiv.org/abs/2503.15752v1), 2025-03-20

(76) [KoGNER: A Novel Framework for Knowledge Graph Distillation on Biomedical Named Entity Recognition](http://arxiv.org/abs/2503.15737v1), 2025-03-19

(77) [Reinforcement Learning Environment with LLM-Controlled Adversary in D&D 5th Edition Combat](http://arxiv.org/abs/2503.15726v1), 2025-03-19

(78) [Leveraging MoE-based Large Language Model for Zero-Shot Multi-Task Semantic Communication](http://arxiv.org/abs/2503.15722v1), 2025-03-19

(79) [Am I eligible? Natural Language Inference for Clinical Trial Patient Recruitment: the Patient's Point of View](http://arxiv.org/abs/2503.15718v1), 2025-03-19

(80) [Safety Aware Task Planning via Large Language Models in Robotics](http://arxiv.org/abs/2503.15707v1), 2025-03-19

(81) [GASP: Unifying Geometric and Semantic Self-Supervised Pre-training for Autonomous Driving](http://arxiv.org/abs/2503.15672v1), 2025-03-19

(82) [ECO: An LLM-Driven Efficient Code Optimizer for Warehouse Scale Computers](http://arxiv.org/abs/2503.15669v1), 2025-03-19

(83) [Enhancing Pancreatic Cancer Staging with Large Language Models: The Role of Retrieval-Augmented Generation](http://arxiv.org/abs/2503.15664v1), 2025-03-19

(84) [R$^2$: A LLM Based Novel-to-Screenplay Generation Framework with Causal Plot Graphs](http://arxiv.org/abs/2503.15655v1), 2025-03-19

(85) [Vision-Speech Models: Teaching Speech Models to Converse about Images](http://arxiv.org/abs/2503.15633v1), 2025-03-19

(86) [LLaVA-MORE: A Comparative Study of LLMs and Visual Backbones for Enhanced Visual Instruction Tuning](http://arxiv.org/abs/2503.15621v1), 2025-03-19

(87) [Does Context Matter? ContextualJudgeBench for Evaluating LLM-based Judges in Contextual Settings](http://arxiv.org/abs/2503.15620v1), 2025-03-19

(88) [SWEET-RL: Training Multi-Turn LLM Agents on Collaborative Reasoning Tasks](http://arxiv.org/abs/2503.15478v1), 2025-03-19

(89) [Cube: A Roblox View of 3D Intelligence](http://arxiv.org/abs/2503.15475v1), 2025-03-19

(90) [From 1,000,000 Users to Every User: Scaling Up Personalized Preference for User-level Alignment](http://arxiv.org/abs/2503.15463v1), 2025-03-19

(91) [Evaluating Bias in Retrieval-Augmented Medical Question-Answering Systems](http://arxiv.org/abs/2503.15454v1), 2025-03-19

(92) [SkyLadder: Better and Faster Pretraining via Context Window Scheduling](http://arxiv.org/abs/2503.15450v1), 2025-03-19

(93) [Visual Position Prompt for MLLM based Visual Grounding](http://arxiv.org/abs/2503.15426v1), 2025-03-19

(94) [Probing the topology of the space of tokens with structured prompts](http://arxiv.org/abs/2503.15421v1), 2025-03-19

(95) [Real-world validation of a multimodal LLM-powered pipeline for High-Accuracy Clinical Trial Patient Matching leveraging EHR data](http://arxiv.org/abs/2503.15374v1), 2025-03-19

(96) [EfficientLLaVA:Generalizable Auto-Pruning for Large Vision-language Models](http://arxiv.org/abs/2503.15369v1), 2025-03-19

(97) [SemEval-2025 Task 1: AdMIRe -- Advancing Multimodal Idiomaticity Representation](http://arxiv.org/abs/2503.15358v1), 2025-03-19

(98) [SPILL: Domain-Adaptive Intent Clustering based on Selection and Pooling with Large Language Models](http://arxiv.org/abs/2503.15351v1), 2025-03-19

(99) [TruthLens:A Training-Free Paradigm for DeepFake Detection](http://arxiv.org/abs/2503.15342v1), 2025-03-19

(100) [Uncertainty-Guided Chain-of-Thought for Code Generation with LLMs](http://arxiv.org/abs/2503.15341v1), 2025-03-19
